extension_helpers/_setup_helpers.py

```diff
@@ -104,6 +104,31 @@ def get_extensions(srcdir="."):
         )
         ext_modules.append(ext)
 
+    # Since https://github.com/astropy/extension-helpers/pull/67,
+    # extensions that used absolute paths in source names stopped working.
+    # Absolute paths in source paths are undesirable but we need to
+    # preserve backward-compatibility until we bump the major release,
+    # so we check for the case of absolute paths and emit a deprecation
+    # warning for now.
+    for extension in ext_modules:
+        sources = []
+        fixed = []
+        for source in extension.sources:
+            if os.path.isabs(source):
+                source = os.path.relpath(source)
+                fixed.append(source)
+            sources.append(source)
+        if fixed:
+            log.warn(
```

Comment:
```suggestion
            log.warning(
```
So as not to undo the (small) benefit from #67 :)
Also, the message should be lazily formatted, but that's anecdotal.